encord/orm/base_dto/base_dto_pydantic_v2.py

```diff
@@ -66,3 +68,10 @@ def from_dict(cls: Type[T], d: Dict[str, Any]) -> T:
 
     def to_dict(self, by_alias=True, exclude_none=True) -> Dict[str, Any]:
         return self.model_dump(by_alias=by_alias, exclude_none=exclude_none, mode="json")  # type: ignore[attr-defined]
+
+
+def dto_validator(mode: str = "before") -> Callable:
```

Comment:
```suggestion
def dto_validator(mode: Literal["before", "after"] = "before") -> Callable:
```
Requires an import of literal of course.